### PR TITLE
✨ Use static props to render events

### LINKED ISSUE #20: ✨Use static props to render events

### PR DESCRIPTION
**Pull request by shawshankkumar**
closes #20

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -164,3 +166,11 @@ const Home: React.FC<HomeProps> = () => {
 };
 
 export default Home;
+
+export async function getStaticProps() {
+  const { data, error } = await supabase.from("config").select("eventName");
```
Comment (sbiswas2209) [inline pages/index.tsx:171]: Avoid database related tasks in frontend as general practice

**Review by sbiswas2209 — CHANGES_REQUESTED** [review]
Create another serverless function and load the event names from there.

File: pages/index.tsx (hunk 1 above)
Comment (shawshankkumar) [reply pages/index.tsx]: Understood.

**Comment by shawshankkumar** [comment]
@sbiswas2209 I have shifted the database calls to an API route and I am using that API route to first fetch and then pass the data as props to the index page function.

**Comment by shawshankkumar** [comment]
@sbiswas2209 I removed the API route(internal routes don't work during the build time) and am now making the database calls inside the statisprops(since it is server side rendered) itself.

**Review by sbiswas2209 — APPROVED** [review]

**Pull request merged**